### PR TITLE
ISSD-52[Learn]  Create site initializer for learn

### PR DESCRIPTION
**Pull request by Andrefes565**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/expando-columns.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			"hidden": false,
+			"indexType": 0,
+			"localizeField": false,
+			"localizeFieldName": true,
+			"secret": false,
+			"visibleWithUpdatePermission": false,
+			"width": "0"
+		}
+	}
+]
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/expando-columns.json:49]: please sort by name

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/liferay-learn/fragments/a/configuration.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"fieldSets": [
+	]
+}
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/liferay-learn/fragments/a/configuration.json:4]: do you really need this?

Comment (nNilton) [reply modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/liferay-learn/fragments/a/configuration.json:4]: I think you can delete everything inside fragments/a/

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by Andrefes565** [comment]
ci:test

**Comment by Andrefes565** [comment]
ci:test:sf

**Comment by Andrefes565** [comment]
ci:test:relevant

**Pull request closed without merging**